src/test/java/mate/academy/validation/PasswordValidatorTest.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.validation;
+
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.model.dto.UserRegistrationDto;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class PasswordValidatorTest {
+    private static final String PASSWORD = "12345678";
+    private static final String INVALID_PASSWORD = "123456789";
```

Comment:
Please think about naming. Looks like this is not invalid password it just repeated password not equalt to the password.